packages/expo-modules-test-core/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import { getAllExpoModulesInWorkingDirectory } from './getStructure';
+import { printModules } from './mockgen';
+
+const command = process.argv[2];
+
+if (command === 'mockgen') {
```

Comment:
I like that you call it `mockgen`, but for the user something clear and simple might be better (e.g. `generate-mocks` or `generate-js-mocks`)